scripts/spi_flash/spi_flash.py

```diff
@@ -989,6 +989,16 @@ def sdcard_upload(self):
         return sd_chksm
 
     def verify_flash(self, req_chksm, old_dictionary, req_dictionary):
+        if bool(self.board_config.get('skip_verify', False)):
+            output("This board requires a manual reboot to")
+            output_line(" complete the flash process.")
+            output("If the board's bootloader uses SDIO mode")
+            output_line(" for its SDCard, then a full")
+            output("power cycle is required.  Please perform")
+            output_line(" the power cycle now and then")
+            output("rerun this utility with the 'check' option")
+            output_line(" to verify flash.")
```

Comment:
It might be more clear to call output_line once here:
```python
output_line(
    "This board requires a manual reboot to complete the flash process.\n"
    "If the board's bootloader uses SDIO mode for its SDCard, then a full\n"
    "power cycle is required.  Please perform  the power cycle now and then\n"
    "rerun this utility with the 'check' option  to verify flash."
)
```

Alternatively you could define the message as a  docstring similar to how Klipper defines its
startup messages in `klippy.py`, in which case it isn't necessary to include newline characters.

Reply:
Yes, it certainly would be more clear.  I originally had it as four calls to `output_line`, one per printed line, but the over restrictive sanity checks yelled at me when I opened the PR because the lines were more than 80 characters long.  So, I broke it into the less readable and maintainable lines that I had.

I can certainly change that to the auto-concatenating strings like you show with the single function call, and see if that passes the 80-character limit.  The lines printed are each less than 80-characters, as I tend to keep output lines under that in scripts for readability in the terminal windows, but with the indentation for Python, they might still exceed the 80-character limit in the check.

Reply:
```python
SDIO_WARNING = """
This board requires a manual reboot to complete the flash process.
If the board's bootloader uses SDIO mode for its SDCard, then a full
power cycle is required.  Please perform  the power cycle now and then
rerun this utility with the 'check' option  to verify flash.
"""

class MCUConnection:
...
        ...
        output_line(SDIO_WARNING)
```

Reply:
That works.  Changes have been made and pushed.  It is much easier to read that way.  Though I still think the sanity checks are a bit over aggressive, IMHO.  On the other hand, it would still fit onto punch cards (ah, the good 'ole days of computers when the worst thing you had to worry about was dropping your stack of punch cards and having to spend the rest of the day manually sorting them).